snuba/cli/replacer.py

```diff
@@ -76,7 +79,7 @@ def replacer(replacements_topic, consumer_group, bootstrap_server, clickhouse_ho
         client_settings=client_settings,
     )
 
-    replacer = BatchingKafkaConsumer(
+    replacer: BatchingKafkaConsumer[Replacement] = BatchingKafkaConsumer(
```

Comment:
😍

Reply:
After the change in PR #481 (https://github.com/getsentry/snuba/pull/481/files#diff-350d7e16635dc2a387158c3fa04de36bR37) you should be able to easily add a type to the snuba ConsumerWorker class as well. Don't know if you want to this together with this change, not blocking on this, it can be done separately as well.

Reply:
I think it'd make sense to handle this afterwards — I'm trying to intentionally limit this to the replacer to reduce risk. (Let me know if it seems like I am misinterpreting this question.)

Reply:
that's fine to me.